marketplace/deployer_util/print_config_test.py

```diff
@@ -35,7 +35,7 @@ def test_load_yaml_file(self):
       f.write("""
               propertyInt: 3
               propertyString: abc
-              """.encode('utf_8'))
```

Comment:
we still need to encode I think, or this file should be opened in text mode with `utf-8` encoding. Or the default of `NamedTemporaryFile` is already text mode?